dist/templates/ovnkube-node.yaml.j2

```diff
@@ -193,6 +193,10 @@ spec:
         - mountPath: /ovn-cert
           name: host-ovn-cert
           readOnly: true
+        - mountPath: /etc/openvswitch/
+          name: host-var-lib-ovs
+        - mountPath: /etc/ovn/
+          name: host-var-lib-ovs
         {% if kind is defined and kind -%}
```

Comment:
@trozet can we set `readOnly: true` for both the mount points?

Reply:
done